src/main/resources/templates/hangoutsNotification.ftl

```diff
@@ -0,0 +1,63 @@
+{
+	"cards": [
```

Comment:
Is this a correct indentation? 8 spaces seems too be very much.

Reply:
It's one tab. It looks like 8 spaces here...